### PR TITLE
Updated for v4

### PR DESCRIPTION
**Pull request by dogversioning**
- updated manifest
- enforced study prefix in it
- corrected some dangling references to system

### CODE REVIEW

File: cumulus_library_suicidality_los/counts.py [hunk 1]
```diff
@@ -6,6 +6,10 @@
 class SuicideLOSCountsBuilder(cumulus_library.CountsBuilder):
     display_text = "Creating suicide LOS counts..."
 
+    def __init__(self, study_prefix: str | None = None):
+        super().__init__()
+        self.study_prefix = "suicide_los"
```
Comment (mikix) [inline cumulus_library_suicidality_los/counts.py:11]: Why does this need to be manually specified?

File: cumulus_library_suicidality_los/version.sql [hunk 2]
```diff
@@ -1,2 +1,2 @@
 CREATE TABLE suicide_los__meta_version AS
-SELECT 2 AS data_package_version;
+SELECT 3 AS data_package_version;
```
Comment (mikix) [inline cumulus_library_suicidality_los/version.sql:2]: Why bump this?

**Review by mikix — APPROVED** [review]

File: cumulus_library_suicidality_los/version.sql (hunk 2 above)
Comment (dogversioning) [reply cumulus_library_suicidality_los/version.sql:2]: good call, reverting

File: cumulus_library_suicidality_los/counts.py (hunk 1 above)
Comment (dogversioning) [reply cumulus_library_suicidality_los/counts.py:11]: I think this is an issue with changes in how these are being loaded with the new approach. I'm going to investigate, but for these older studies, hard setting it should take care of this problem in the meantime.

**Pull request merged**